third_party/sl4a/build.gradle

```diff
@@ -36,3 +52,81 @@ artifacts {
     archives javadocJar
     archives sourcesJar
 }
+
+// bintray packaging information
+install {
+    repositories.mavenInstaller {
+        pom.project {
+            name ARTIFACT_ID
+            description
+                'Library for triggering device-side code from host-side ' +
+                'Mobly tests.'
+            url 'https://github.com/google/mobly-snippet-lib'
+
+            packaging 'aar'
+            groupId GROUP_ID
+            artifactId ARTIFACT_ID
+            version VERSION_NAME
+            name ARTIFACT_ID
+
+            licenses {
+                license {
+                    name 'The Apache Software License, Version 2.0'
+                    url 'http://www.apache.org/licenses/LICENSE-2.0.txt'
+                    distribution 'repo'
+                }
+            }
+            scm {
+                connection 'https://github.com/google/mobly-snippet-lib.git'
+                url 'https://github.com/google/mobly-snippet-lib'
+            }
+        }
+    }
+}
+
+// Try loading bintray.properties. It's okay if this fails; we might not be
+// uploading.
+def bintrayPropFile = project.rootProject.file('bintray.properties')
+if (bintrayPropFile.exists()) {
+    Properties prop = new Properties()
+    prop.load(new FileInputStream(bintrayPropFile))
+    project.ext.bintrayUser = prop.user
+    project.ext.bintrayKey = prop.key
+} else {
+    project.ext.bintrayUser = null
+    project.ext.bintrayKey = null
+}
+
+gradle.taskGraph.whenReady { graph ->
+    if (graph.hasTask(bintrayUpload)) {
+        if (project.bintrayUser == null || project.bintrayKey == null) {
+            if (bintrayPropFile.exists()) {
+                throw new IllegalArgumentException(
+                    bintrayPropFile.toString()
+                    + " does not contain 'user' or 'key' props")
+            } else {
+                throw new FileNotFoundException(
+                    "bintray.properties not found at path: " + bintrayPropFile)
+            }
+        }
+    }
+}
+
+bintray {
+    user = project.bintrayUser
+    key = project.bintrayKey
+    configurations = ['archives']
+    pkg {
+        repo = 'mobly'
```

Comment:
To clarify: bintray has the concept of 'repo' and 'package'.
The 'repo' is mobly and can be used for additional mobly-related artifacts in the future.
The first and currently only 'package' in this repo is 'mobly-snippet-lib'.